oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java

```diff
@@ -1306,9 +1306,13 @@ private void updateCoords(String cluster, BundleJob bundle,
                 delay = getDelay((Feed) entity, coord);
             }
 
-            //calculate next start time based on delay.
-            endTime = (delay == null) ? endTime
-                    : EntityUtil.getNextStartTime(coord.getStartTime(), delay, EntityUtil.getTimeZone(entity), endTime);
+            //calculate end time based on delay only for feed with delay.
```

Comment:
@sandeepSamudrala  Can you explain how start time of new coord will be calculated